src/Http/Http.Extensions/gen/StaticRouteHandlerModel/EndpointParameter.cs

```diff
@@ -109,10 +109,7 @@ private void ProcessEndpointParameterSource(Endpoint endpoint, ISymbol symbol, I
             }
             // Complex form binding is only supported in RDF because it uses shared source with Blazor that requires dynamic analysis
             // and codegen. Emit a diagnostic when these are encountered to avoid producing buggy code.
```

Comment:
```suggestion
            // Minimal APIs shares the same implementation that Blazor uses for complex form binding which uses reflection.
            // We could implement a source gen layer for form binding but the plan has been to do that in conjunction with Blazor,
            // so we're not solving the same problem twice in different ways.
```

I couldn't follow this. Can we just rewrite it to match [your PR comment](https://github.com/dotnet/aspnetcore/pull/55465#discussion_r1586881468) on the PR that added it?

Reply:
Ah, good note. After re-reading this, I realize it's not particularly helpful for understanding the reasoning. 😅 

Tweaked the verbiage here a bit.